crates/ide/src/goto_definition.rs

```diff
@@ -125,6 +130,62 @@ pub(crate) fn goto_definition(
     Some(RangeInfo::new(original_token.text_range(), navs))
 }
 
+// If the token is into(), try_into(), parse(), search the definition of From, TryFrom, FromStr.
+fn find_from_definition(
```

Comment:
The name is rather confusing / non-descriptive so I'd rather have a longer one like (especially given this handles more than just `From` impls
```suggestion
fn find_definition_for_known_blanket_dual_impls(
```